ext/crypto/lib.deno_crypto.d.ts

```diff
@@ -237,21 +237,132 @@ declare var CryptoKeyPair: {
  * @category Crypto
  */
 interface SubtleCrypto {
+  /**
+   * The SubtleCrypto.generateKey() method is part of the Web Cryptography API, which allows web applications to perform cryptographic operations. This method is used to generate a cryptographic key.
+   *
+   * Use Cases
+   * 
+   * - Generating symmetric keys for encryption/decryption.
+   * - Creating public/private key pairs for asymmetric encryption.
+   * - Ensuring secure communication in web applications.
+   * 
+   * @example
+   * ```ts
+   * // RSA key generation
+   * const key = await.crypto.subtle.generateKey(
```

Comment:
typo

Reply:
oh thanks!
took me _ages_ to spot what my editor had done to me there!